scripts/globals/job_utils/beastmaster.lua

```diff
@@ -0,0 +1,132 @@
+-----------------------------------
+-- beastmaster Job Utilities
+-----------------------------------
+
+-----------------------------------
+xi = xi or {}
+xi.job_utils = xi.job_utils or {}
+xi.job_utils.beastmaster = xi.job_utils.beastmaster or {}
```

Comment:
No need for the extra gaps in here, unless you have some requires:

```lua
-----------------------------------
-- Warrior Job Utilities
-----------------------------------
xi = xi or {}
xi.job_utils = xi.job_utils or {}
xi.job_utils.warrior = xi.job_utils.warrior or {}
```

```lua
-----------------------------------
-- Thief Job Utilities
-----------------------------------
require('scripts/globals/quests')
require('scripts/globals/utils')
-----------------------------------
xi = xi or {}
xi.job_utils = xi.job_utils or {}
xi.job_utils.thief = xi.job_utils.thief or {}

```

Reply:
Thanks for the feedback, I have fixed the PR.